### DISCUSSION TITLE
Popup States ausblenden wenn Datenwert = "false"

### DISCUSSION THREAD

**Discussion by tschortschde** [mcuiobroker/docuJarvis#26] [Q&A]
Hallo,
ist es möglich im Popup States zu filtern bzw. auszublenden?
Ich möchte gerne die geschlossenen Fenster im Popup nicht anzeigen...
![Screenshot 2023-03-24 195339](https://user-images.githubusercontent.com/11274233/227614693-41032d9d-eb14-4100-ae4f-17688da187c3.jpg)

VG

**Comment by mcuiobroker** [comment]
Nein. -> Feature Request bei jarvis.

evtl. mit scripts möglich. Muss man dann aber halt programmieren, dass der DP bei dem Zustand ausgeblendet werden soll.

**Comment by tschortschde** [comment]
Funktioniert mit 
https://github.com/Zefau/ioBroker.jarvis/issues/1509
